src/main/java/cz/cvut/kbss/study/service/security/SecurityUtils.java

```diff
@@ -102,7 +99,10 @@ private User resolveAccountFromOAuthPrincipal(Jwt principal) {
             throw new NotFoundException(
                     "User with username '" + userInfo.getPreferredUsername() + "' not found in repository.");
         }
-        roles.stream().map(Role::forName).filter(Optional::isPresent).forEach(r -> user.addType(r.get().getType()));
+
+        RoleGroup roleGroup = new RoleGroup();
```

Comment:
I'm not sure if this will work if we do not have a concrete role name here for internal-auth, at least.

Reply:
it is only from keycloak